android/src/main/res/values/breakdown2016_styles.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="breakdown_base">
```

Comment:
Try it without the `app:` namespace; that's how I created the toolbar default style.